sublime-package-types.json

```diff
@@ -0,0 +1,111 @@
+{
+  "contributions": {
```

Comment:
The base schema for LSP packages should ideally be in the LSP package itself but I think there is no way to do that currently. I mean, to contribute a generic, pattern-less schema that can be referenced by other schemas. But should be possible to support fairly easily.

BTW. Creating schemas by hand is a bit of a pain. :)